yarn.lock

```diff
@@ -108,10 +108,10 @@
   resolved "https://registry.npmjs.org/@internetarchive/field-parsers/-/field-parsers-0.1.3.tgz"
   integrity sha512-gN7ivwHWlf8V7fD3VakQ4NtddSou5i3A3TqbIo8EKF+XTxJXaXpNBmzkZEGf2oMVK8rHvrabFxZKVEy9qG7Omw==
 
-"@internetarchive/histogram-date-range@^0.1.8":
-  version "0.1.9"
-  resolved "https://registry.npmjs.org/@internetarchive/histogram-date-range/-/histogram-date-range-0.1.9.tgz"
-  integrity sha512-jXcdh7nmVxSjcAfKrMLtpEty3BNL3Y1RLenQWg44x7EpLXtAFcwRDdnMFq5SDCZqOU+tqt9cTK6IPfsWfhNkEQ==
+"@internetarchive/histogram-date-range@^0.1.10-alpha.1":
+  version "0.1.10-alpha.1"
+  resolved "https://registry.yarnpkg.com/@internetarchive/histogram-date-range/-/histogram-date-range-0.1.10-alpha.1.tgz#13df5bc3e4a086d11d96bdbf854c8ef2c7a97d60"
```

Comment:
how come we have npmjs registry and yarnpkg registry here?

Reply:
Huh, I'm not sure why that's happening. Good eyes! I'll see if another quick `yarn install` fixes things...

Reply:
Had to explicitly tell yarn to use the npmjs registry, but should be fixed now. That said, it probably makes little difference which registry gets used (I think they both pull from the same place in the end anyway).